instrumentation/src/splunk.h

```diff
@@ -3,15 +3,22 @@
 
 #include <stdbool.h>
 #include "logger.h"
+#include "config.h"
 
-static char *const DISABLE_ENV_VAR_NAME = "DISABLE_SPLUNK_AUTOINSTRUMENTATION";
+static char *const conf_file = "/usr/lib/splunk-instrumentation/splunk.conf";
```

Comment:
Do we need to include a default or sample `splunk.conf` in the deb/rpm packages?

Also, we should include some documentation about this file and the supported keys/values.

Reply:
Yeah, we'll likely want to include a config file in the packages and for sure document everything -- didn't want to do that in this PR as I wanted to wait until how this is going to work gets firmed up. This change is incremental to keep PRs small and work easier to track on the way towards the full solution, which will have the service name set properly by reading the CLI args, and possibly have the install script set the service name pattern. Do you think we should put this into a feature branch instead of main?

Reply:
I think it should be fine to merge this to main since we're not releasing this yet.  I'm working on adding this to the release pipeline (#1236), but will wait until until we are ready.